vars/runPipeline.groovy

```diff
@@ -234,6 +230,17 @@ def testDownstreamProject (name) {
                     '''
                     break;
 
+                case 'dlang-community/libdparse':
+                    sh 'cd test && DMD=$DC ./run_tests.sh'
```

Comment:
We do expose both DMD and DC in the pipeline, just as the install.sh script does.